examples/pytorch/ogb/ogbn-mag/hetero_rgcn.py

```diff
@@ -365,13 +371,15 @@ def test(g, model, node_embed, y_true, device, split_idx):
 
     return train_acc, valid_acc, test_acc
 
+def is_support_affinity(v_t):
+    return v_t >= "0.9.1"
```

Comment:
@chang-l , you're right, I found it is 0.9.0 DataLoader is not supporting 'enable_cpu_affinity' yet, modified check func here